<launch>
    <!--  pick object service node -->
    <node name="plan_grasps_service" pkg="tams_pick_and_place_pointnetgpd" type="plan_grasps_service">

        <param name="bound_frame" value="table_top" />

        <!-- Size of the bound determining if a grasp is valid -->
        <param name="x_bound" value="0.60" />
        <param name="y_bound" value="0.80" />
        <param name="z_bound" value="0.50" />

        <!-- Offset of the bound from the origin of the frame -->
        <param name="x_bound_offset" value="0.0" />
        <param name="y_bound_offset" value="0.0" />
        <param name="z_bound_offset" value="0.29" />

        <!-- Offset of the gripper endeffector link from the detected grasp  should be -0.079 -->
        <param name="grasp_offset" value="-0.06" />

        <!-- grasps older than this value in seconds won't be considered anymore -->
        <param name="grasp_cache_time_threshold" value="5.0" />

        <param name="move_group_arm" value="arm" />
        <param name="move_group_gripper" value="gripper" />
    </node>

    <!--  pick object demo node -->
    <node name="pick_object_demo" pkg="tams_pick_and_place_pointnetgpd" type="pick_object_demo">
        <param name="move_group_arm" value="arm" />
        <param name="move_group_gripper" value="gripper" />
    </node>
</launch>
